src/com/rcv/ResultsWriter.java

```diff
@@ -185,33 +184,57 @@ public void generateSummarySpreadsheet() {
     org.apache.poi.ss.usermodel.Row eliminationsRow = worksheet.createRow(rowCounter++);
     // eliminationsRowHeader is the header cell for the eliminations row
     Cell eliminationsRowHeader = eliminationsRow.createCell(0);
-    eliminationsRowHeader.setCellValue("Candidates defeated");
+    eliminationsRowHeader.setCellValue("Candidate(s) defeated");
+
+    // Winner -- display is shifted to subsequent round for display
```

Comment:
I think most of the changes you made to ResultsWriter are redundant with changes I already made in this commit a couple weeks ago: https://github.com/BrightSpots/rcv/commit/3a468ee7626b3010b86f51df9c21074c23de473c